CHANGELOG/6.0.md

```diff
@@ -736,7 +736,7 @@ For more information on this, we invite you to read [this blog post explaining P
 - Once the pipeline is running as a job, all of the standard `*-Job` cmdlets can be used to manage the job.
 - Variables (ignoring process-specific variables) used in the pipeline are automatically copied to the job so `Copy-Item $foo $bar &` just works.
 - The job is also run in the current directory instead of the user's home directory.
-- For more information about PowerShell jobs, see [about_Jobs](https://msdn.microsoft.com/powershell/reference/6/about/about_jobs).
+- For more information about PowerShell jobs, see [about_Jobs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_jobs?view=powershell-6).
```

Comment:
```suggestion
- For more information about PowerShell jobs, see [about_Jobs](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_jobs?view=powershell-6).
```

Reply:
I suggest to remove "?view=powershell-6" suffix too.

Reply:
This is the change-log doc for 6.0, so I guess it's OK to have that suffix.